src/app/payment-status/payment-status.component.scss

```diff
@@ -0,0 +1,37 @@
+.status-form {
+  display: grid;
+
+  button {
+    width: 138px;
+    margin: 56px auto;
+    font-size: 16px;
+    background-color: #624de3;
+    font-family: Inter, sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    line-height: 20px;
+    color: #ffffff;
```

Comment:
Please update the height of the button
![image](https://user-images.githubusercontent.com/59651445/102644672-dec8b880-4169-11eb-8d8c-bd7ee0472b44.png)